metadata.rb

```diff
@@ -10,15 +10,15 @@
 
 recipe 'kafka::default', 'Downloads and installs Kafka from binary releases'
 
-suggests 'java', '~> 1.22' # ~FC052
+depends 'java', '~> 1.22'
```

Comment:
This cookbook doesn't actually depend on the `java` cookbook, it just requires that there's a working Java installation present, so people are free to manage that how they see fit.

While `suggests` doesn't actually _do_ anything, I added it as a "sign" that Java must be present and working. Perhaps we should just do away with it completely and make sure that the README states that a working Java installation is assumed. What do you think?